pagebuilder/publish.go

```diff
@@ -43,7 +43,8 @@ func (p *Page) GetPublishActions(db *gorm.DB, ctx context.Context, storage oss.S
 		Content:  content,
 		IsDelete: false,
 	})
-	p.SetOnlineUrl(p.getPublishUrl(localePath, categoryPath))
+	dir, _ := path.Split(p.getPublishUrl(localePath, categoryPath))
+	p.SetOnlineUrl(dir)
```

Comment:
@moluuser 
I think the complete path (including the file name) should be stored in the db.
You can split it where it shows

Reply:
OK, thank you for your review.